### ISSUE TITLE
Fabric: Spawn point in wrong place

### ISSUE THREAD

**Issue by ProClifo** [webbukkit/dynmap#3860]
> template is **bold**
> sample data is *italicized*

**Issue Description:** *In the Nether and End, the globe icon that shows Spawn is in the wrong place. They are using the same spawn point as the overworld. The reason for this is Fabric/Vanilla servers do not have a separate world spawn points in the same way Paper servers do. On paper, you can set the world spawn for each dimension individually. In Vanilla/Fabric, there is only one spawn point for the server, because no one would ever respawn naturally in the End or Nether in vanilla gameplay unless they have a respawn anchor set. Dynmap uses that same spawn point for all three dimensions. On my server, where I have set the overworld spawn point to 37000 blocks away from 0,0, the problem becomes much more apparent. The only sensible thing would be to have End spawn at the obsidian spawn platform (at 100,48,0) and Nether spawn at overworld spawn coordinates divided by 8.*

* **Dynmap Version:** *Dynmap-3.4-fabric-1.19.1*
* **Server Version:** *fabric 1.19.2*
* **Pastebin of Configuration.txt:** *https://pastebin.com/Pc2T24GH*

* **Other Relevant Data/Screenshots:**
* ![image](https://user-images.githubusercontent.com/8874666/194658026-983c9b6d-c894-44de-92d1-237acd6086eb.png)
* ![image](https://user-images.githubusercontent.com/8874666/194658187-fbc5f774-d644-4452-809f-9d8b46b226d3.png)
* ![image](https://user-images.githubusercontent.com/8874666/194653559-43662f2a-fece-4f3a-9747-f2a29b37e14a.png)
* This is my personal Dynmap where the problem is present: http://murucraft.xyz:8123

* **Steps to Replicate:**
* 1. Make a Fabric server with Dynmap installed
* 2. Set the world spawn point in the overworld
* 3. Watch as the other two dimensions change world spawn too

 [x] *I have looked at all other issues and this is not a duplicate*  
 [x] *I have been able to replicate this*

**Comment by kosma** [comment]
Yeah, it's a known bug. I'm hoping to fix it next time I sit down to work on the Fabric port... thanks for reporting!
